boards/arm/imxrt/imxrt1060-evk/src/imxrt_flexcan.c

```diff
@@ -0,0 +1,96 @@
+/****************************************************************************
+ * boards/arm/imxrt/imxrt1060-evk/src/imxrt_flexcan.c
+ *
+ *
+ * Redistribution and use in source and binary forms, with or without
```

Comment:
Please change the copright to apache.

Reply:
@xiaoxiang781216 Sure. Just by adding
 *   Copyright (C) 2020 Gregory Nutt. All rights reserved.
 *   Author: Michal Lenc <lencmich@fel.cvut.cz>
 *
 * Based on: arch/arm/src/s32k1xx/s32k1xx_flexcan.h

or is there a need to change the whole Apache license as in [Coding Standard](https://cwiki.apache.org/confluence/display/NUTTX/Coding+Standard#cfilestructure)? Thanks.

Reply:
Yes, it needs to be changed to Apache license

Reply:
@acassis Thanks, it´s done.